modules/administration/pages/recurring-actions.adoc

```diff
@@ -0,0 +1,55 @@
+[[recurring-actions]]
+= Recurring Actions
+
+
+You can schedule automated recurring highstate actions for Salt clients.
+Recurring highstate actions apply the highstate to clients on a specified schedule.
+You can apply recurring action to individual clients, to all clients in a system group, or to an entire organization.
+
+
+[IMPORTANT]
+====
+Recurring actions can only be used with Salt clients.
+You cannot schedule recurring actions on traditional clients.
+====
+
+
+
+.Procedure: Creating a New Recurring Action
+. To apply a recurring action to an individual client, navigate to [guimenu]``Systems``, click the client to configure schedules for, and navigate to the menu:States[Recurring States] tab.
+. To apply a recurring action to a system group, navigate to menu:Systems[System Groups], select the group to configure schedules for, and navigate to menu:States[Recurring States] tab.
+. Click btn:[Create].
+. Type a name for the new schedule.
+. Choose the frequency of the recurring action:
++
+* [guimenu]``Hourly:`` Type the minute of each hour.
+For example, [parameter]``15`` will run the action at fifteen minutes past every hour.
+* [guimenu]``Daily:`` Select the time of each day.
+For example, [parameter]``01:00`` will run the action at 0100 every day, in the timezone of the {productname} Server.
+* [guimenu]``Weekly:`` Select the day of the week and the time of the day, to execute the action every week at the specified time.
+* [guimenu]``Monthly:`` Select the day of the month and the time of the day, to execute the action every month at the specified time.
+* [guimenu]``Custom Quartz format:`` For more detailed options, enter a custom quartz string.
+For example, to run a recurring action at 0215 every Saturday of every month, enter:
```

Comment:
I think this is Sunday.  My CRONTAB(5) says:
```
day of week    0-7 (0 or 7 is Sunday, or use names)
```
```suggestion
For example, to run a recurring action at 0215 every Sunday of every month, enter:
```

Reply:
`0 15 2 ? * 7` is: `Every Saturday at 02:15`
So it's right as it is right now